options.js

```diff
@@ -445,13 +451,20 @@ function showChattyNewsSettings(){
     ele.checked = getOption("chatty_news_highlight_article_posts");
     ele = document.getElementById('chatty_news_wallpaper')
     ele.checked = getOption("chatty_news_show_image");
-    var scrollPref = getOption("enabled_scripts").contains("scrolling_performance_hack");
+    var scrollPref = getOption("enabled_scripts").indexOf("scrolling_performance_hack") > -1;
```

Comment:
You can also leave out scrolling performance hack stuff. It's not needed on the current version of Chatty, and it was deprecated as of version 1.5x, IIRC.

Reply:
Sure, I'll remove that.